WBooks/AddNew/AddNewViewController.swift

```diff
@@ -23,11 +23,110 @@ class AddNewViewController: UIViewController {
     
     override func viewDidLoad() {
         super.viewDidLoad()
+        setupActions()
         navigationController?.navigationBar.topItem?.title = NSLocalizedString("TITLE_ADDNEW", comment: "")
     }
     
     override func loadView() {
         view = addNewView
     }
+    
+    override func viewDidAppear(_ animated: Bool) {
+        addNewView.submitButton.withGradient()
+        setupTextField()
+    }
+    
+    func setupTextField(){
+        addNewView.bookNameTextField.config()
+        addNewView.authorTextField.config()
+        addNewView.yearTextField.config()
+        addNewView.topicTextField.config()
+        addNewView.descriptionTextField.config()
+    }
```

Comment:
Is this method necessary ? I see that the init of CustomTextField has the config

Reply:
I call the textfield configuration again, since in the view it is done but with the size that the textfield has so far, so when changing the screen it does not adapt, speaking with Diego, he told me that a possible solution is to call it in the viewDidAppear to take the size of the textfield after executing the constraints